File: Cameras.vb
Public Class Owner
    Public Property url As String
    Public Property id As Integer
    Public Property email As String
    Public Property first_name As String
    Public Property last_name As String
    Public Property updated_at As DateTime
End Class

Public Class Location
    Public Property type As String
    Public Property coordinates As Double()
End Class

Public Class Camera
    Public Property url As String
    Public Property id As Integer
    Public Property serial_number As String
    Public Property owner As Owner
    Public Property name As String
    Public Property timezone As String
    Public Property status As Integer
    Public Property bulb_on As Boolean
    Public Property alarm_on As Boolean
    Public Property led_mask As Boolean
    Public Property bluetooth_identifier As String
    Public Property updated_at As DateTime
    Public Property recordings_url As String
    Public Property users_url As String
    Public Property sensitivity As Integer
    Public Property build As Integer
    Public Property volume As Integer
    Public Property notifications_enabled As Boolean
    Public Property doorbell_press_notifications_enabled As Boolean
    Public Property location As Location
    Public Property location_address As String
    Public Property subscription As Object
    Public Property dawn_offset As Integer
    Public Property dusk_offset As Integer
    Public Property motion_timeout As Integer
    Public Property mesh_group_id As Integer
    Public Property companions_count As Integer
    Public Property down_at As DateTime
    Public Property sight_option As Integer
    Public Property sight_stationary_filter As Boolean
    Public Property sight_stationary_mse_filter As Boolean
    Public Property play_msg_on_detect As Integer
    Public Property created_at As DateTime
    Public Property sight_on_lite As Boolean
    Public Property ip_address As String
    Public Property recording_active As Boolean
    Public Property brightness As Integer
    Public Property video_flip As Boolean
    Public Property sight_on As Boolean
    Public Property sight_on_override As Boolean
    Public Property autosight As Boolean
    Public Property sight_permissions As Integer
    Public Property server_up As String
    Public Property support_permission_end As Object
    Public Property custom_messages As Object()
    Public Property chime_duration_use_default As Boolean
    Public Property chime_duration As Double
    Public Property on_cam_ai As Boolean
End Class

Public Class Cameras
    Public Property count As Integer
    Public Property [next] As Object
    Public Property previous As Object
    Public Property results As Camera()
End Class

File: Cameras.cs
using System;


    public class Owner
    {
        public string url { get; set; }
        public int id { get; set; }
        public string email { get; set; }
        public string first_name { get; set; }
        public string last_name { get; set; }
        public DateTime updated_at { get; set; }
    }

    public class Location
    {
        public string type { get; set; }
        public double[] coordinates { get; set; }
    }


    public class Camera
    {
        public string url { get; set; }
        public int id { get; set; }
        public string serial_number { get; set; }
        public Owner owner { get; set; }
        public string name { get; set; }
        public string timezone { get; set; }
        public int status { get; set; }
        public bool bulb_on { get; set; }
        public bool alarm_on { get; set; }
        public bool led_mask { get; set; }
        public string bluetooth_identifier { get; set; }
        public DateTime updated_at { get; set; }
        public string recordings_url { get; set; }
        public string users_url { get; set; }
        public int sensitivity { get; set; }
        public int build { get; set; }
        public int volume { get; set; }
        public bool notifications_enabled { get; set; }
        public bool doorbell_press_notifications_enabled { get; set; }
        public Location location { get; set; }
        public string location_address { get; set; }
        public object subscription { get; set; }
        public int dawn_offset { get; set; }
        public int dusk_offset { get; set; }
        public int motion_timeout { get; set; }
        public int mesh_group_id { get; set; }
        public int companions_count { get; set; }
        public DateTime down_at { get; set; }
        public int sight_option { get; set; }
        public bool sight_stationary_filter { get; set; }
        public bool sight_stationary_mse_filter { get; set; }
        public int play_msg_on_detect { get; set; }
        public DateTime created_at { get; set; }
        public bool sight_on_lite { get; set; }
        public string ip_address { get; set; }
        public bool recording_active { get; set; }
        public int brightness { get; set; }
        public bool video_flip { get; set; }
        public bool sight_on { get; set; }
        public bool sight_on_override { get; set; }
        public bool autosight { get; set; }
        public int sight_permissions { get; set; }
        public string server_up { get; set; }
        public object support_permission_end { get; set; }
        public object[] custom_messages { get; set; }
        public bool chime_duration_use_default { get; set; }
        public double chime_duration { get; set; }
        public bool on_cam_ai { get; set; }
    }

    public class Cameras
    {
        public int count { get; set; }
        public object next { get; set; }
        public object previous { get; set; }
        public Camera[] results { get; set; }
    }

